plugin/neovim-fuzzy.vim

```diff
@@ -208,16 +212,18 @@ function! s:fuzzy(choices, opts) abort
       return
     endif
 
-    let result = readfile(self.outputs)
-    if !empty(result)
-      let file = self.handler(result)
-      exe 'lcd' self.root
-      silent execute g:fuzzy_opencmd fnameescape(expand(file.name))
-      lcd -
-      if has_key(file, 'lnum')
-        silent execute file.lnum
-        normal! zz
-      endif
+    let results = readfile(self.outputs)
+    if !empty(results)
+      for result in results
+        let file = self.handler([result])
+        exe 'lcd' self.root
+        silent execute g:fuzzy_opencmd fnameescape(expand(file.name))
```

Comment:
When `g:fuzzy_opencmd` is `tab drop` then it's great to be able to open multiple matches like this, when `opencmd` is set to `:edit` it won't be so useful.